src/__tests__/searchAlgorithm.test.js

```diff
@@ -0,0 +1,19 @@
+import searchAlgorithm from '../utils/searchAlgorithm'
+describe('searchAlgorithm', () => {
+  it('returns filtered results when input and data are provided', () => {
+    const input = 'mass'
+    const colleges = [{ name: 'Massachusetts Institute of Technology (MIT)', country: 'United States', city: 'Cambridge' }]
```

Comment:
Let's reuse the` colleges` in all these three tests. So, let's initialize it inside the describe.